layouts/partials/head/meta.html

```diff
@@ -27,10 +27,8 @@
 {{ end }}
 
 
-{{ if .Site.RSSLink -}}
-<link href="{{ .Site.RSSLink }}" rel="alternate" type="application/rss+xml" title="{{ .Site.Title }}" />
-<link href="{{ .Site.RSSLink }}" rel="feed" type="application/rss+xml" title="{{ .Site.Title }}" />
-{{- end }}
+<link href="{{ .Site.BaseURL }}blog/index.xml" rel="alternate" type="application/rss+xml" title="{{ .Site.Title }}" />
+<link href="{{ .Site.BaseURL }}blog/index.xml" rel="feed" type="application/rss+xml" title="{{ .Site.Title }}" />
```

Comment:
Usually, we would use `{{ "blog/index.xml" | absURL }}` instead of constructing the URL ourselves from the base URL.

I know they amount to the same thing but I would prefer to keep this consistent unless this approach doesn't work here for some reason.

Reply:
should be ok now